examples/text_to_image/train_text_to_image.py

```diff
@@ -138,7 +139,9 @@ def save_model_card(
         f.write(yaml + model_card)
 
 
-def log_validation(vae, text_encoder, tokenizer, unet, args, accelerator, weight_dtype, epoch):
+def log_validation(
+    vae, args, text_encoder, tokenizer, accelerator, weight_dtype, epoch, unet=None, is_final_validation=False
+):
```

Comment:
```suggestion
def log_validation(
    unet, vae, text_encoder, tokenizer, args, accelerator, weight_dtype, epoch, is_final_validation=False
):
```

Let's keep the signature mentally more readable.